src/actions/AddSiteActions.js

```diff
@@ -8,9 +12,24 @@ import {
     SITE_NEAREST_TOWN_TEXT_CHANGED,
     SITE_ACCESSIBILITY_OPTION_CHANGED,
     SITE_FACILITIES_OPTION_CHANGED,
-    ADD_SITE_FIELDS_RESET
+    ADD_SITE_FIELDS_RESET,
+    ADD_SITE_SUCCESS,
+    ADD_SITE_FAILURE,
+    CHECK_IF_SITE_IS_READY
 } from './types';
 
+export const openSiteUploadModal = () => {
+    return {
+        type: OPEN_SITE_UPLOAD_MODAL
+    }
+};
+
+export const closeSiteUploadModal = () => {
+    return {
+        type: CLOSE_SITE_UPLOAD_MODAL
+    }
+};
```

Comment:
@amgoodman5 Finally took care of putting these on global state instead of local state. I needed to do it so we could close the modal after success/failure of an upload.

Reply:
Awesome